entities/card.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+from entities.card_type import CardDesignProperties
+
+
+@dataclass
+class Card:
+    content: str
+    type: CardDesignProperties
```

Comment:
I dont like the type of the card is define with content and design stuff, if you need values for the design stuff, we can define card with:
content: str
design_properties: CardDesignProperties
metadata: CardMetadata